src/CatCard/CatCard.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Image,
+  Text,
+  View,
+} from 'react-native';
+
+import { styles } from './styles';
+
+export const CatCard = ({ cat }) => (
+  <View style={styles.catCard}>
+    <View style={styles.headInfo}>
+      <Text style={styles.name}>{cat.name}</Text>
+      <Text style={styles.breed}>{cat.breed}</Text>
+    </View>
+    <View style={styles.divider} />
+    <Image source={require('../catImg/angora.png')} style={styles.img} />
+    <Text style={styles.description}>{cat.description}</Text>
+  </View>
+);
+
+CatCard.propTypes = {
+  cat: PropTypes.objectOf(PropTypes.string).isRequired,
```

Comment:
As I see `cat` object has `name`, `breed` fields and possibly `image`
In such case this way `PropTypes.objectOf(PropTypes.string)` is a good option because each field has `string` type. At the same type `objectOf` is not descriptive enough for e newbie. Better to describe all fields through `shape` prop type